pipeline/ffmpeg.go

```diff
@@ -129,15 +129,13 @@ func (f *ffmpeg) HandleStartUploadJob(job *JobInfo) (*HandlerOutput, error) {
 	sourceSegments := sourceManifest.GetAllSegments()
 	job.sourceSegments = len(sourceSegments)
 
-	if job.sourceSegments > 0 && job.InputFileInfo.Format != "hls" {
-		firstSeg := sourceSegments[0]
-		lastSeg := sourceSegments[job.sourceSegments-1]
+	if job.sourceSegments > 1 && job.InputFileInfo.Format != "hls" {
+		segmentsToCheck := []int{0, 1, job.sourceSegments - 2, job.sourceSegments - 1}
```

Comment:
Do we need range checks to catch e.g a single segment video?

Reply:
Yeah right now it just wouldn't enter the if-statement, will change that